### ISSUE TITLE
need a proper documentation of Readme.md

### ISSUE THREAD

**Issue by Arman-ali-khan-786** [Arman-ali-khan-786/calculator#5]

**Comment by utkarsh006** [comment]
Kindly assign it to me.

**Comment by Arman-ali-khan-786** [comment]
> Kindly assign it to me.

yeah sure and make sure to make a pr after hacktoberfest has officially start other it will not be counted

**Comment by utkarsh006** [comment]
@Arman-ali-khan-786 It will be starting from tomorrow isn't it?

**Comment by Arman-ali-khan-786** [comment]
> @Arman-ali-khan-786 It will be starting from tomorrow isn't it?

Yes

**Comment by himanshu251001** [comment]
@Arman-ali-khan-786 assign me

**Comment by Arman-ali-khan-786** [comment]
> @Arman-ali-khan-786 assign me

yeah work on it and update the contributors list like adding a api that fetches automatically the list of names

**Comment by utkarsh006** [comment]
@Arman-ali-khan-786 I have already created PR for the issue. Merge the PR. @himanshu251001 Buddy kindly make new issue else you will face merge conflicts. Happy Contributing :)

**Comment by Arman-ali-khan-786** [comment]
> > @Arman-ali-khan-786 assign me
> 
> yeah work on it and update the contributors list like adding a api that fetches automatically the list of names

you also work on it
